scripts/ci/travis_build.sh

```diff
@@ -24,7 +24,7 @@ if [ -n "${TRAVIS_TAG}" ]; then
 
 elif [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
   ARCH_NUMBER=$(arch_to_build_number ${ARCH})
-  echo "Building dev (nightly)"
+  echo "Building alpha (nightly)"
```

Comment:
We might use alphas in the regular release process as well. I wouldn't change this.